scRNA-seq/pre-processing_scRNA-seq.md

```diff
@@ -0,0 +1,149 @@
+# Pre-processing single cell RNA-seq data 
+
+**CCDL 2019**
+
+#### In this section, we will be running through the basics of pre-processing
+#### single-cell RNA-seq data.
+
+For the purposes of this tutorial, we'll summarize single-cell technologies as
+being one of two groups, based on their capture methods and quantitative nature.
+Depending on how the cells are sorted and what technology is used, the pre-processing steps are a bit different and the biases to look out for in post-processing also vary.
+
+For a more extensive run through on single-cell technologies,
+Kiselev at al have very [good tutorial for scRNA-seq](https://hemberg-lab.github.io/scRNA.seq.course/introduction-to-single-cell-rna-seq.html#experimental-methods).
+
+### 1) Non-Tag-Based scRNA-seq  
+*Example:* Smart-seq2 [(Picelli et al, 2014)](https://www.nature.com/articles/nprot.2014.006)   
+Cells are physically separated generally into individual wells in a plate and
+often also sorted by other means (eg. Fluorescence Activated Cell Sorting).
+Each cell is then sequenced individual and has it's own fastq file (this will be two fastq files if this is paired-end sequencing.)
+The data pre-processing steps for these types of scRNA-seq data is/can be more similar to
+bulk rna-seq methods.
```

Comment:
Doesn't look like this change has been made as I'm viewing it on Github.
```suggestion
bulk RNA-seq methods.
```

Reply:
Yeah, I remember having already made this change. But maybe I'm thinking of a different module? Oh well, I've remade the changes here.